x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/RestListConnectorAction.java

```diff
@@ -43,13 +43,15 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
         List<String> connectorNames = List.of(restRequest.paramAsStringArray("connector_name", new String[0]));
         List<String> serviceTypes = List.of(restRequest.paramAsStringArray("service_type", new String[0]));
         String searchQuery = restRequest.param("query");
+        Boolean isDeleted = restRequest.paramAsBoolean("include_deleted", false);
```

Comment:
Nit: the variable name `isDeleted` and the param pulled `include_deleted` don't really match, I'm a bit confused about which is being fetched (list of connectors that have been deleted, vs list of connectors including those that are soft deleted)

Reply:
Good spot, yeah it should be `Boolean includeDeleted = ...` (leftover after the rename)

Reply:
Fixed https://github.com/elastic/elasticsearch/pull/118669/commits/d2700e8611b13ccd6764df9cbd90f4e114e23f39